src/accelerate/utils/modeling.py

```diff
@@ -316,6 +317,8 @@ def set_module_tensor_to_device(
                 device = f"musa:{device}"
             elif is_xpu_available():
                 device = f"xpu:{device}"
+            elif is_hpu_available():
+                device = "hpu"
```

Comment:
the modeling code relies on safetensors to load tensors so for now I'm installing safetensors from main to have HPU support https://github.com/huggingface/safetensors/pull/566